README.md

```diff
@@ -86,124 +96,195 @@ See the playground to interactively use this:
 - the server does not loose it's data (e.g., due to a broken hard-drive)
 
 We call this "stay" that lasts as long as the above criteria are met a session.
 
 Below we describe:
+
 1. [Session data](#session-data)
-    - [Client-side access](#client-side-access)
-    - [Server-side access](#server-side-access)
+   - [Client-side access](#client-side-access)
+   - [Server-side access](#server-side-access)
 2. [How to configure session-storage](#storage-backends)
 3. [Configuration](#configuration)
 4. [Security](#security)
 
 ### Session Data
 
-The session that `nuxt-session` maintains for you also allows you to store arbitrary data in a storage across requests for the entire duration of the session. `nuxt-session` makes this data available to you in different ways, depending on whether you are on the client side (e.g., `.vue` components that are seen by your users) or on the server-side (e.g., inside an endpoint in the `server/api/` directory).
+The session that `nuxt-session` maintains for you also allows you to store
+arbitrary data in a storage across requests for the entire duration of the
+session. `nuxt-session` makes this data available to you in different ways,
+depending on whether you are on the client side (e.g., `.vue` components that
+are seen by your users) or on the server-side (e.g., inside an endpoint in the
+`server/api/` directory).
 
-Reading session data is generally safe on both the client- and server-side, unless it contains anything you don't want your users to see.
+Reading session data is generally safe on both the client- and server-side,
+unless it contains anything you don't want your users to see.
 
-Allowing alteration of session-data with arbitrary data provided by the client (e.g., your user) should be treated carefully, but can be safely done if you don't care about your users polluting sessions, have authentication and authorization or are generally not concerned about the security of your app.
+Allowing alteration of session-data with arbitrary data provided by the client
+(e.g., your user) should be treated carefully, but can be safely done if you
+don't care about your users polluting sessions, have authentication and
+authorization or are generally not concerned about the security of your app.
 
 #### Client Side Access
 
 On the client-side you can use the session like this:
+
 ```ts
 const {
   session,
   refresh,
   remove,
   reset,
   update,
-  overwrite
-} = await useSession()
+  overwrite,
+} = await useSession();
 
 // The session itself, a ref that automatically updates when you use the other methods below
-session.value
+session.value;
 
 // Refresh the session, e.g., after you've changed the session on the server side OR when you don't have an active session at the moment
-await refresh()
+await refresh();
 
 // Delete the current session without getting a new one. Note that the next request will automatically get a new session
-await remove()
+await remove();
```

Comment:
also all of these changes: let's leave them out of the PR - we don't want to have a big PR just due to auto formatting

Reply:
ugh, sorry. I keep forgetting to disable my auto format when I work on this :facepalm: